lib/DDG/Spice/Cryptocurrency.pm

```diff
@@ -142,10 +158,13 @@ sub checkCurrencyCode {
 
     # If both currencies are available, use the ticker endpoint
     if (length($from) && length($to)) {
-        # Return early if both currencies are in the excluded list
+        # Return early if both currencies are in the excluded list uncless a generic trigger
+        # which will not trigger the currency spice
         # Allows searches like "ftc to aud" but excludes searches like "aud to usd"
-        if (exists($excludedCurrencies{$from}) && exists($excludedCurrencies{$to})) {
-            return;
+        if(!$generic) {
+            if (exists($excludedCurrencies{$from}) && exists($excludedCurrencies{$to})) {
+                return;
```

Comment:
This can be simplified: `return if !generic && (exists($excludedCurrencies{$from}) && exists($excludedCurrencies{$to}))`.

You could breakup the line on the first `&&` for legibility if you wanted